Exiled.API/Features/Toys/AdminToy.cs

```diff
@@ -0,0 +1,91 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="AdminToy.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Toys
+{
+    using AdminToys;
+    using Exiled.API.Enums;
+    using Mirror;
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for <see cref="AdminToys.AdminToyBase"/>.
+    /// </summary>
+    public abstract class AdminToy
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AdminToy"/> class.
+        /// </summary>
+        /// <param name="toyAdminToyBase">The <see cref="AdminToys.AdminToyBase"/> to be wrapped.</param>
+        /// <param name="type">The <see cref="AdminToyType"/> of the object.</param>
+        public AdminToy(AdminToyBase toyAdminToyBase, AdminToyType type)
+        {
+            AdminToyBase = toyAdminToyBase;
+            ToyType = type;
+        }
+
+        /// <summary>
+        /// Gets the original <see cref="AdminToys.AdminToyBase"/>.
+        /// </summary>
+        public AdminToyBase AdminToyBase { get; }
```

Comment:
Everything in EXILED uses a `Base` word, I would use it here too

Reply:
No wait, not this but in `Light` class

Reply:
oh wait, this is the admintoy class, since its derivated from it we cant use the same name

Reply:
So if you have a light `li` you can get `li.Base` and `li.AdminToyBase`